### PR TITLE
Waffle #3

### PR DESCRIPTION
**Pull request by concach**
spec 2 more tests passing, none additional failing, modified seeds to establish business-items

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ def self.start
     seed = Seed.new
     seed.generate_categories
     seed.generate_businesses
+    seed.generate_business_items
     seed.generate_users
-    seed.generate_items
```
Comment (karinamzalez) [inline db/seeds.rb]: was this name change simply to clarify the association between items and business?

**Comment by kbs5280** [comment]
Ok to merge. No issues or additional comments.
